webapp/platform/types/src/client4.ts

```diff
@@ -3,8 +3,6 @@
 
 export enum LogLevel {
     Error = 'ERROR',
-    Warning = 'WARNING',
-    Info = 'INFO',
```

Comment:
These ones in the `types` package are exported for use outside of MM or because they match up with server-side APIs